### PR TITLE
#257 add an auto reveal checkbox, for the session admin

### PR DESCRIPTION
**Pull request by SponsoredByPuma**
Gave the admin the option to automatic reveal the votes, once everybody voted or to wait for the timer to come to an end. While the Estimations are still running, you cannot change the mode.

### CODE REVIEW

**Comment by stritti** [comment]
Could you also update user docs for this feature?

**Review by stritti — APPROVED** [review]

File: backend/src/main/java/io/diveni/backend/controller/WebsocketController.java [hunk 1]
```diff
@@ -183,6 +197,28 @@ public synchronized void processVote(@Payload String vote, MemberPrincipal membe
     LOGGER.debug("<-- processVote()");
   }
 
+  @MessageMapping("/vote/autoreveal")
```
Comment (stritti) [inline backend/src/main/java/io/diveni/backend/controller/WebsocketController.java:200]: The endpoints are confusing:
here: `/vote/autoreveal`
in line 153 `/startVoting/autoreveal`

Do not use captial letters in uri. Probably we could make more descriptive endpoints?

**Review by stritti — CHANGES_REQUESTED** [review]

**Comment by KzuDemEvin** [comment]
Why is this not supported for planning without user stories?

**Comment by SponsoredByPuma** [comment]
HI @KzuDemEvin,
It should also work for NO_US mode now. I also removed some unnecessary functions.

**Review by KzuDemEvin — DISMISSED** [review]

**Comment by stritti** [comment]
Merge Conflicts should be fixed

File: frontend/src/locales/de.json
Comment (stritti) [inline frontend/src/locales/de.json:1]: translations should be done within Crowdin. See translation guide: https://sybit-education.github.io/Diveni/guide/translations

**Review by stritti — CHANGES_REQUESTED** [review]

**Pull request merged**